app/controllers/comments_controller.rb

```diff
@@ -534,4 +534,12 @@ def redirect_to_all_comments(commentable, options = {})
                   :page => options[:page]
     end
   end
+
+  private
+
+  def comment_params
+    params.require(:comment).permit(
+      :pseud_id, :content, :name, :email, :edited_at, :unreviewed
```

Comment:
If you don't mind my asking, why is `:unreviewed` included as one of the whitelisted params? It's currently marked as `attr_protected`, so as I understand it, it couldn't have been included in any previous mass assignments, and probably shouldn't be included in future mass assignments. Am I missing something? (Is there a test that requires it?)